# This message is for storing segmentation masks in a harvester format
float64[] x  # Array of [x1 x2 ... xn] in image co-ordinates
float64[] y  # Array of [y1 y2 ... yn] in image co-ordinates
float64[] z  # Array of [z1 z2 ... zn] in image co-ordinates
int16 class_id